### ISSUE TITLE
Not excluding .bbclass files, oelint-adv 7.2.4

### ISSUE THREAD

**Issue by DaniilKl** [priv-kweihmann/oelint-adv#729]
I want to exclude some of my classes. But adding regex to global `exclude`  or `oelint-adv` `files` does not work for me:

```diff
diff --git a/.pre-commit-config.yaml b/.pre-commit-config.yaml
index f7bd2c021802..155916db09b0 100644
--- a/.pre-commit-config.yaml
+++ b/.pre-commit-config.yaml
@@ -2,7 +2,7 @@
 default_stages: [pre-commit]
 
 default_install_hook_types: [pre-commit, commit-msg]
-exclude: .*\.(patch)|(diff)$|CHANGELOG.md
+exclude: .*\.(patch)|(diff)$|CHANGELOG.md|^(meta-zarhus-distro/classes-recipe/)|^(meta-zarhus-distro/recipes-devtools/rust/)
 
 repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
@@ -51,7 +51,7 @@ repos:
         entry: oelint-adv
         language: python
         language_version: python3
-        files: .*\.(bb)|(bbappend)|(bbclass)$
+        files: .*\.(bb)|(bbappend)|(bbclass)$|^(meta-zarhus-distro/classes-recipe/)|^(meta-zarhus-distro/recipes-devtools/rust/)
 
   - repo: https://github.com/codespell-project/codespell
     rev: v2.3.0
```

Output:

```bash
Advanced oelint..........................................................Failed
- hook id: oelint-adv
- exit code: 1
- files were modified by this hook

/home/danillklimuk/Projects/MyProj/meta-zarhus/meta-zarhus-distro/classes-recipe/cargo.bbclass:13:warning:oelint.var.multiinherit:'rust-target-config' is included multiple times [branch:true]
/home/danillklimuk/Projects/MyProj/meta-zarhus/meta-zarhus-distro/classes-recipe/cargo.bbclass:45:warning:oelint.vars.spacesassignment:Suggest spaces around assignment. E.g. 'FOO = "BAR"' [branch:true]
/home/danillklimuk/Projects/MyProj/meta-zarhus/meta-zarhus-distro/classes-recipe/rust-common.bbclass:1:error:oelint.bbclass.underscores:bbclass filenames should not contain '-'. Replace it by '_' [branch:true]
/home/danillklimuk/Projects/MyProj/meta-zarhus/meta-zarhus-distro/classes-recipe/rust-common.bbclass:18:error:oelint.vars.appendop:Use ':append' instead of ' += ' as it overwrites 'RUSTFLAGS ??= ""
' [branch:true]
/home/danillklimuk/Projects/MyProj/meta-zarhus/meta-zarhus-distro/classes-recipe/rust-target-config.bbclass:1:error:oelint.bbclass.underscores:bbclass filenames should not contain '-'. Replace it by '_' [branch:true]
```
